.github/workflows/pre-commit.yml

```diff
@@ -12,3 +12,15 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
       - uses: pre-commit/action@v3.0.0
+
+      - name: Install clojure tools
+        uses: DeLaGuardo/setup-clojure@12.3
+        with:
+          clj-kondo: 2022.05.31
+          cljfmt: 0.10.2
+
+      - name: Run clj-kondo
+        run: clj-kondo --lint frontend/src
+
+      - name: Run cljfmt
+        run: cljfmt check frontend/src
```

Comment:
I can see that cljs-kondo has support for pre-commit hooks. Could you please add hook in https://github.com/clj-kondo/clj-kondo/blob/master/doc/ci-integration.md#pre-commit-framework to `.pre-commit-config.yaml` so we use only pre-commit hooks?

Reply:
seems like running it as container is working locally without having clj-kondo installed on my machine

```
  - repo: https://github.com/clj-kondo/clj-kondo
    rev: v2023.12.15
    hooks:
      - id: clj-kondo-docker
```

hook passes on my machine

Reply:
@nikromen can you please let me know what is the motivation for using the hooks as suggested? I am a bit lost in the nuances of GitHub Actions.

Reply:
```
diff --git a/.pre-commit-config.yaml b/.pre-commit-config.yaml
index 96d1a53..9d3582a 100644
--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -42,3 +42,8 @@ repos:
         additional_dependencies:
           - types-PyYAML
           - types-requests
+
+  - repo: https://github.com/clj-kondo/clj-kondo
+    rev: v2023.12.15
+    hooks:
+      - id: clj-kondo-docker
```